.github/workflows/is-accl-release-build.yml

```
@@ -0,0 +1,71 @@
+
+      - name: Set up JDK 21
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: '21'
+
+      # Extract the version from the pom.xml file so it can be reused
+      # in the tag name, release name, and uploaded asset filename
+      - name: Extract version from pom.xml
+        id: extract_version
+        run: |
```

actionlint (1.7.12)

[error] workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)